Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -80,6 +80,18 @@ void CleanupFromPreviousUpdate()
 #else
   File::Delete(reloc_updater_path);
 #endif
+
+#ifdef __APPLE__
+  std::string embedded_updater_path =
+    File::GetBundleDirectory() + "/Contents/Helpers/" + UPDATER_FILENAME;
```

Comment:
const

Both here and TriggerUpdate need the Updater path so let's make a function for that.